src/platform-includes/enriching-events/breadcrumbs/automatic-breadcrumbs/react-native.mdx

```diff
@@ -0,0 +1,91 @@
+The React Native SDK already captures breadcrumbs automatically via the Native SDKs for Android and iOS.
+
+- [Automatic Breadcrumbs for Android](/platforms/android/enriching-events/breadcrumbs/#automatic-breadcrumbs)
+- [Automatic Breadcrumbs for iOS](/platforms/apple/configuration/integrations/default/#sentryautobreadcrumbtrackingintegration)
+
+- For interaction breadcrumbs wrapp your `App` using the `Sentry.wrap` method.
+
+```javascript
+import * as React from 'react';
+import { Text, View } from 'react-native';
+import * as Sentry from '@sentry/react-native';
+
+Sentry.init({
+  // parameters
+});
+
+const App = () => {
+  return (
+    <View>
+      <Text>Hello World</Text>
+    </View>
+  );
+};
+
+export default Sentry.wrap(App);
+```
+
+- For navigation breadcrumbs use one of the SDK's natigation integrations.
+    - `Sentry.ReactNativeNavigationInstrumentation`
+    - `Sentry.ReactNavigationInstrumentation`
+    - `Sentry.ReactNavigationV4Instrumentation`
+
+```javascript
+import * as React from 'react';
+import { Text, View } from 'react-native';
+import {
+  NavigationContainer,
+  NavigationContainerRef,
+} from '@react-navigation/native';
+import * as Sentry from '@sentry/react-native';
+
+const instrumentation = new Sentry.ReactNavigationInstrumentation();
+
+Sentry.init({
+  // Other parameters
+  integrations: [
+    new Sentry.ReactNativeTracing({
+      routingInstrumentation: instrumentation,
+    }),
+  ],
+});
+
+const App = () => {
+  const navigation = React.useRef<NavigationContainerRef>();
+
+  return (
+    <NavigationContainer
+      ref={navigation}
+      onReady={() =>
+        instrumentation.registerNavigationContainer(navigation)}>
+      {/* Navigator and screens */}
+    </NavigationContainer>
+  );
+};
+```
+
+- To track `HTTP` requests enable tracing either by `tracesSampler` or `tracesSampleRate` and `enableAutoPerformanceTracking` options. Or setup your own `Sentry.RectNativeTracing`.
+
+```javascript
+import * as Sentry from '@sentry/react-native';
+
+Sentry.init({
+  tracesSampleRate: 1.0,
+  enableAutoPerformanceTracking: true,
+  // Other parameters
+});
+```
+
+- To track redux actions and latest store state use `Sentry.createReduxEnhancer`.
```

Comment:
Does this work for RN? I've never tested myself.
Worth adding to https://docs.sentry.io/platforms/react-native/configuration/integrations

This page does not exist yet, ideally, it should, with all the possible integrations.
Example https://docs.sentry.io/platforms/dart/configuration/integrations/

Reply:
Yes it does, it’s in the sample app. Ive tested it.

Reply:
It's actually from JS SDK, so I'll just add link to it.